pb/config/config.proto

```diff
@@ -2,6 +2,8 @@ syntax = "proto3";
 
 // Protocol buffer for configuring testgrid.k8s.io
 
+// NOTE: Do NOT update this until you have updated the other config.proto!
```

Comment:
Is there anything we can put to decrease the ambiguity of "the other"?

Reply:
Changed to 'internal' to be clearer. ^^